package.json

```diff
@@ -32,13 +32,13 @@
   },
   "homepage": "https://github.com/Automattic/eslint-config-wpvip#readme",
   "dependencies": {
-    "@babel/eslint-parser": "7.21.3",
+    "@babel/eslint-parser": "^7.22.10",
     "@rushstack/eslint-patch": "1.2.0",
-    "@typescript-eslint/eslint-plugin": "5.55.0",
-    "@typescript-eslint/parser": "5.55.0",
+    "@typescript-eslint/eslint-plugin": "^5.55.0 || ^6.0.0",
+    "@typescript-eslint/parser": "^5.55.0 || ^6.0.0",
```

Comment:
Why the `||`? these aren't peerDeps, so they will resolve the left hand side of the operator and will be stuck there via `package-lock`. (In this case, `@typescript-eslint/parser` gets `5.62`)

Reply:
This was because of `eslint-plugin-jest` :-( It should usually resolve to the highest possible version.

Reply:
Updated to `6.0.0` and `eslint-plugin-jest` from 27.2.1 to 27.1.3.